dat/ships/hyena.xml

```diff
@@ -13,7 +13,7 @@
  <trail_generator x="-15" y="-6" h="0">fire-thin</trail_generator>
  <trail_generator x="-15" y="6" h="0">fire-thin</trail_generator>
  <fabricator>Aerosys</fabricator>
- <description>The star product of Aerosys, the Hyena quickly has also become a favourite of guerrilla and paramilitary groups far and wide. Their top-notch thrust combined with fast reaction time allow them exhaust enemy energy supplies while wearing them down. While Aerosys claim that their ships are only sold to reputable pilots, it's suspected that they sell to anyone willing to pay, as it seems there are countless Hyenas among the ranks of the more notorious and unruly pilots.</description>
+ <description>The star product of Aerosys, the Hyena quickly has also become a favourite of guerrilla and paramilitary groups far and wide. Their top-notch thrust combined with fast reaction time allow them to exhaust enemy energy supplies while wearing them down. While Aerosys claim that their ships are only sold to reputable pilots, it's suspected that they sell to anyone willing to pay, as it seems there are countless Hyenas amoung the ranks of the more notorious and unruly pilots.</description>
```

Comment:
Is this `amoung` correct? I don't thing the spelling `amoung` is used at all these days, even in British English.

Reply:
I stand corrected... apologies, I've been under the impression it was "amoung" and "amongst" - it seems that even the, largely hidden, British dictionaries are against me on this! (OED can go jump however, contrarian scholars that they are!)